### PR TITLE
[ZEPPELIN-1033] HotFixing of paragraph deletion and re-ordering broken

### PR DESCRIPTION
**Pull request by astroshim**
### What is this PR for?
This PR is for fixing [this issue](https://issues.apache.org/jira/browse/ZEPPELIN-1033)

### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1033

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by randerzander** [comment]
@astroshim still getting same errors. After trying to "move up" and then "delete" a cell, I get these errors in the log: http://pastebin.com/Sr4WveXF

**Comment by astroshim** [comment]
@randerzander Thank you for your review. Let me check.

**Comment by astroshim** [comment]
@randerzander Did you build this PR? 
You can get this PR from ```git clone https://github.com/astroshim/zeppelin -b ZEPPELIN-1033```.
My zeppelin is working well as following screenshot.
test

Could you please check one more?

**Comment by AhyoungRyu** [comment]
It works as before 👍

File: zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java [hunk 1]
```diff
@@ -21,6 +21,9 @@
 import org.apache.shiro.subject.Subject;
 import org.apache.shiro.util.ThreadContext;
 import org.apache.shiro.web.mgt.DefaultWebSecurityManager;
+import org.apache.shiro.mgt.SecurityManager;
+import org.apache.shiro.config.Ini;
```
Comment (khalidhuseynov) [inline zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java]: may need to remove import of `Ini`, doesn't seem to be used

File: zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java [hunk 2]
```diff
@@ -34,6 +37,12 @@
  */
 public class SecurityUtils {
 
+  public static void initSecurityManager(String shiroPath) {
+    IniSecurityManagerFactory factory = new IniSecurityManagerFactory("file:" + shiroPath);
+    SecurityManager securityManager = factory.getInstance();
+    org.apache.shiro.SecurityUtils.setSecurityManager( securityManager );
```
Comment (khalidhuseynov) [inline zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java]: maybe remove spaces

**Comment by khalidhuseynov** [comment]
I checked it, and behavior with paragraph reordering fixed. So basically, previously shiro was initialized only when needed from `shiro.ini`, and now it's initialized always when starting the server explicitly. to be honest, i'm not exactly sure at the moment about the best way to solve the issue with `SecurityManager` initialization, since previously it was also initialized (automatically through the classpath) but wasn't available (possibly because of not injecting correctly). But definitely this one way to solve the issue at the moment, which can be further improved

File: zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java (hunk 1 above)
Comment (astroshim) [inline zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java]: Thank you for your review. I removed importing Ini.

File: zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java (hunk 2 above)
Comment (astroshim) [inline zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java]: Thank you for your review. I removed spaces.

**Comment by randerzander** [comment]
I tried again from your branch and it worked. I must have applied your commit incorrectly last time.. my bad! Thanks for the fix =)

**Comment by astroshim** [comment]
nice @randerzander thanks for addressing this issue and review.

**Comment by Leemoonsoo** [comment]
+1 for merge

**Comment by minahlee** [comment]
Merging it as hotfix

**Pull request closed without merging**